testing/src/test/java/epam/com/FileOutputTestsExtension.java

```diff
@@ -0,0 +1,28 @@
+package epam.com;
+
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.time.LocalDateTime;
+
+import static java.lang.String.format;
+import static java.time.format.DateTimeFormatter.ISO_DATE_TIME;
+
+public class FileOutputTestsExtension implements AfterTestExecutionCallback {
+
+
+  @Override
+  public void afterTestExecution(ExtensionContext extensionContext) throws Exception {
+    try (OutputStream out = new FileOutputStream("src/test/resources/log.txt", true)) {
```

Comment:
I'm getting the following error:
```shell
[INFO] Running epam.com.generator.TemplateGeneratorTest
OpenJDK 64-Bit Server VM warning: Sharing is only supported for boot loader classes because bootstrap classpath has been appended
[ERROR] Tests run: 9, Failures: 0, Errors: 5, Skipped: 1, Time elapsed: 3.004 s <<< FAILURE! -- in epam.com.generator.TemplateGeneratorTest
[ERROR] epam.com.generator.TemplateGeneratorTest.createTemplateInConsoleMode_ProvidedWrongPlaceholderValue_ThrowsException(String)[1] -- Time elapsed: 2.708 s <<< ERROR!
java.io.FileNotFoundException: src/test/resources/log.txt (No such file or directory)
```